### PR TITLE
added num likes 

### PR DESCRIPTION
**Pull request by caramelmelmel**
@gauravkeerthi just need to add the userlikes which is giving an annoying type error

### CODE REVIEW

**Comment by caramelmelmel** [comment]
just remove the package lock shd be fine

**Comment by caramelmelmel** [comment]
@gauravkeerthi have fixed the merge conflict for you

**Comment by gauravkeerthi** [comment]
@joelkek is the check failure because of the package lock file? I'm busy this weekend so will only get to look at why the checks are failing on Sunday night.

File: .gitignore [hunk 1]
```diff
@@ -365,5 +365,6 @@ $RECYCLE.BIN/
 
 # build directory
 build
+SKIP_PREFLIGHT_CHECK=true
```
Comment (gauravkeerthi) [inline .gitignore:368]: I propose to undo this edit. There's no reqmt to skip the preflight checks.

File: src/models/ReflectionLikes.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import { firestore} from '../firebase'
+
+export const createReflectionLikes = (id, uidLike)=>{
+  var washingtonRef = firestore.collection('reflectionResponses').doc(id);
+  washingtonRef.get().then((doc) => {
+    if (doc.exists) {
+        //console.log("Document data:", doc.data().reflectionId);
+        console.log(doc.data().numLikes)
+        console.log()
```
Comment (gauravkeerthi) [inline src/models/ReflectionLikes.js:9]: reminder to self: remove all the console.log calls later.

File: .gitignore (hunk 1 above)
Comment (caramelmelmel) [reply .gitignore:368]: okok

File: src/models/ReflectionLikes.js (hunk 2 above)
Comment (caramelmelmel) [reply src/models/ReflectionLikes.js:9]: oops thank u

**Comment by caramelmelmel** [comment]
have removed all that I can

**Pull request merged**